cpp/ps0003/ps0003_soln.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+
+using namespace std;
+int main()
+{
+  // prompting user for number 2/2
+  int number;
+  cout << "please enter number \n";
+  cin >> number;
```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number